vite.config.js

```diff
@@ -10,7 +10,7 @@ export default defineConfig({
     middlewareMode: 'ssr',
```

Comment:
It's important to run vite in SSR mode here.

Reply:
sample comment should be minimized too.

Reply:
how about this: 

```
const path = require('path');
```

Reply:
Just one more comment to play with.

Reply:
Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a treatise on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.10.32.